libraries/abstractions/transport/utest/CMakeLists.txt

```diff
@@ -1,101 +1,89 @@
-    project ("transportsecure sockets unit test")
-    cmake_minimum_required (VERSION 3.13)
+project ("transportsecure sockets unit test")
+cmake_minimum_required (VERSION 3.13)
 
 # ====================  Define your project name (edit) ========================
-    set(project_name "transport_secure_sockets")
+set(project_name "transport_secure_sockets")
 
 # =====================  Create your mock here  (edit)  ========================
 
 # list the files to mock here
-    list(APPEND mock_list
-                "${kernel_dir}/include/task.h"
-                "${kernel_dir}/include/portable.h"
-                "${common_dir}/include/iot_logging_task.h"
-                "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
-            )
+list(APPEND mock_list
+            "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
+        )
 # list the directories your mocks need
-    list(APPEND mock_include_list
-                "${CMAKE_CURRENT_LIST_DIR}/include"
-            )
+list(APPEND mock_include_list
+            "${CMAKE_CURRENT_LIST_DIR}/include"
```

Comment:
Is this required? There is no `include/` directory in `transport/utest` folder

Reply:
removed in f3c02d76cc5cd6364f37d551a4f8825ff22d64d4